static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-liveblog-world-cup.js

```diff
@@ -0,0 +1,240 @@
+    const inView = elementInView(el, window, {
+        top: 18,
+    });
+
+    inView.on('firstview', () => {
+        logView(test.id);
+        mediator.emit(test.viewEvent);
+    });
+};
+
+const addEpicToBlocks = (epicHtml: string, test: EpicABTest): Promise<void> =>
+    getBlocksToInsertEpicAfter().then(blocksToInsertEpicAfter => {
+        blocksToInsertEpicAfter.forEach(el => {
+            getLiveblogEntryTimeData(el).then((timeData: TimeData) => {
+                fastdom.write(() => {
+                    const $epic = $.create(epicHtml);
+                    $epic.insertAfter(el);
+                    mediator.emit(test.insertEvent);
+                    $(el).removeClass(INSERT_EPIC_AFTER_CLASS);
+                    setEpicLiveblogEntryTimeData($epic[0], timeData);
+                    setupViewTracking(el, test);
+                });
+            });
+        });
+    });
+
+export const setupEpicInLiveblog = (
+    epicHtml: string,
+    test: EpicABTest
+): void => {
+    addEpicToBlocks(epicHtml, test);
+
+    if (!isAutoUpdateHandlerBound) {
+        mediator.on('modules:autoupdate:updates', () => {
+            addEpicToBlocks(epicHtml, test);
+        });
+        isAutoUpdateHandlerBound = true;
+    }
+};
+
+export const acquisitionsEpicLiveblogWorldCup: EpicABTest = makeABTest({
+    id: 'AcquisitionsEpicLiveblogWorldCup',
+    campaignId: 'epic_liveblog_world_cup',
+    campaignSuffix: pageId.replace(/-/g, '_').replace(/\//g, '__'),
+
+    start: '2018-06-21',
+    expiry: '2019-01-24',
+
+    author: 'Jonathan Rankin',
+    description:
+        'This places the epic underneath liveblog blocks which the author has specified in Composer should have an epic against them',
+    successMeasure: 'Member acquisition and contributions',
+    idealOutcome:
+        'Our wonderful readers will support The Guardian in this time of need!',
+
+    audienceCriteria: 'All',
+    audience: 1,
+    audienceOffset: 0,
+
+    pageCheck(page) {
+        return page.contentType === 'LiveBlog' && tagsMatch();
+    },
+
+    variants: [
+        {
+            id: 'control',
+            products: ['CONTRIBUTION', 'MEMBERSHIP_SUPPORTER'],
+
+            options: {
+                isUnlimited: true,
+
+                template(variant) {
+                    return epicLiveBlogTemplate({
+                        copy: liveblogCopy(
+                            variant.options.supportURL,
+                            variant.options.contributeURL
+                        ),
+                        componentName: variant.options.componentName,
+                    });
+                },
+
+                test(renderFn, variant, test) {
+                    const epicHtml = variant.options.template(variant);
+                    setupEpicInLiveblog(epicHtml, test);
+                },
+            },
+        },
+        {
+            id: 'depth',
+            products: ['CONTRIBUTION', 'MEMBERSHIP_SUPPORTER'],
+
+            options: {
```

Comment:
maybe you could create a function local to this module that takes `copy` and returns this object, since all other params are the same between variants